src/Apps/Auction/Routes/Register/index.tsx

```diff
@@ -104,8 +107,17 @@ export const RegisterRoute: React.FC<RegisterProps> = props => {
         },
         onError: reject,
         mutation: graphql`
-          mutation RegisterCreateCreditCardMutation($input: CreditCardInput!) {
-            createCreditCard(input: $input) {
+          mutation RegisterPreCreateBidderMutation(
```

Comment:
So is this name intended to name the sequence of mutations: { `updateMyUserProfile`, `createCreditCard` }? Does the name of  `RegisterPreCreateBidderMutation` follow some existing pattern?

Perhaps that this is just me, but the name of `RegisterPreCreateBidderMutation` really doesn't mean much. In particular, the "pre" is confusing.

Thoughts on a name like `BidderRegistrationMutation`? Maybe the function could also be named `createBidderRegistration` as well for parity? Right now there are lot of terms flowing around: requirements, pre-create bidder, etc – and I think for the sake of clarity it would be useful to reduce them.

Reply:
Before this change we had two mutations:

1. createCreditCard
2. createBidder

We're now updating user profile information in the first mutation. Instead of changing that to something like `createCreditCardAndUpdatePhoneNumber`, we went with the "pre" name which comes before bidder creation.

We already have a `RegisterCreateBidderMutation` defined which executed the `createBidder` behavior. This other mutation is more of a "pre" step, hence the name.

Open to other suggestions though.

Reply:
@dblandin and I chatted briefly IRL and decided that `createCreditCardAndUpdatePhoneNumber` was the least mysterious name for now while requiring minimal further refactorings